templates/api_dump/api.json.njk

```diff
@@ -0,0 +1 @@
+{{ api.dump | safe }}
```

Comment:
This is the new template being added for test purposes. Running the generator with `--template=api_dump` will produce the above JSON representation, which will help debugging weird problems.